pkg/models/results.go

```diff
@@ -82,6 +85,29 @@ func (groupInfo *GroupInfo) IsCalled() bool {
 	return false
 }
 
+func (groupInfo *GroupInfo) IndexString() string {
+	// Assumes IDs is sorted
+	return strings.Join(groupInfo.IDs, ",")
+}
+
+// FixedVersions returns a list of fixed versions, or an empty slice if no fixed version is available
```

Comment:
Comment seems wrong? We're returning a map right?

Reply:
Correct it in: 5f1fd7a471c91a6065a92814389ab6399a399bee